testsuite/MDAnalysisTests/datafiles.py

```diff
@@ -372,6 +373,7 @@
 TPR_xvf = (_data_ref / 'cobrotoxin.tpr').as_posix()
 TRR_xvf = (_data_ref / 'cobrotoxin.trr').as_posix()
 H5MD_xvf = (_data_ref / 'cobrotoxin.h5md').as_posix()
+H5MD_energy = (_data_ref / 'cu.h5').as_posix()
```

Comment:
is it the convention for other `h5md` files out there to use the `*.h5` extension rather than `*.h5md`?

Reply:
I can rename the file to `*.h5md`. I've seen both and personally prefer `*.h5` because I'm lazy and did not configure the VS Code H5WEB extension to also read `*.h5md`. Afaik H5MD is the only format specification for H5 files, so there is no risk of confusion.